internal/jobobject/oci.go

```diff
@@ -0,0 +1,17 @@
+package jobobject
```

Comment:
I don't think the file should be named oci/this package should mention oci at all. I know we use them to convert to the notion of a "cpu count" that OCI works with but I don't think these are only applicable for that. However, I don't know what would be a good name here 😆 I trust your naming prowess, it's better than mine

Reply:
I see your point and I agree. The original idea was that since this file converts between the OCI spec value of cpu limits (iow processor weight and count) to job object relative versions, it was oci related.

Reply:
Updated. What do you think?

Reply:
I don't mind utilities, I know Kevin had some reservations about it as it can lead to it becoming a dumping ground for things. Maybe conversions.go?